CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
-## v0.7.18 - 2023-09-18
+## v0.7.18 - 2023-09-18 (aborted release/not recommended)
```

Comment:
It might be good to add a special note on why this was aborted, e.g.

```
<!--changelog-parser-ignore-start-->
Note: This release was aborted due to a regression in the log sharing functionality
<!--changelog-parser-ignore-end-->
```